greymatter-io/gitops-core
# Permissioning Redis to receive Prometheus health checks

[sc-25164]

## How to test

0. Build the operator from `main`
1. Branch off of this submodule and set spire=true in inputs.cue
2. Point the operator at your branch off of this one
3. Launch the operator and confirm that Prometheus' health checks appear in the dashboard of the resulting mesh once everything is up.
